src/pyproject_toml.rs

```diff
@@ -98,10 +168,24 @@ impl PyProjectToml {
     }
 
     /// Returns the value of `[tool.maturin.sdist-include]` in pyproject.toml
+    #[deprecated(
+        since = "0.14.0",
+        note = "please use `PyProjectToml::include` (<https://github.com/PyO3/maturin/pulls/1255>)"
+    )]
```

Comment:
maturin is mostly used as a cli, so this `deprecated` annotation won't be noticed by users.